docs/v3/api-ref/cli/artifact.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: " "
+sidebarTitle: prefect artifact
+---
+
+
+# `prefect main artifact`
+
+
+
+```command
+prefect main artifact [OPTIONS] COMMAND [ARGS]...
```

Comment:
seems like it's picking up part of the import path and incorrectly appended that to the CLI; the command should be `prefect artifact`